### PR TITLE
Pass `clerkJSVersion` in interstitial

### PR DESCRIPTION
**Pull request by panteliselef**
## Type of change

- [ ] 🐛 Bug fix
- [x] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [x] `@clerk/nextjs`
- [x] `@clerk/remix`
- [ ] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/localizations`
- [ ] `@clerk/clerk-expo`
- [x] `@clerk/backend`
- [x] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/fastify`
- [ ] `@clerk/chrome-extension`
- [ ] `gatsby-plugin-clerk`
- [ ] `build/tooling/chore`

## Description

Review per commit. (Changes for each package)

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.

### CODE REVIEW

**Comment by panteliselef** [comment]
!snapshot

**Comment by clerk-cookie** [comment]
Hey @panteliselef - the snapshot version command generated the following package versions:
| Package | Version |
| --- | --- |
| @clerk/backend | 0.23.1-snapshot.fe3ca7b |
| @clerk/chrome-extension | 0.3.9 |
| @clerk/clerk-js | 4.47.0 |
| eslint-config-custom | 0.3.0 |
| @clerk/clerk-expo | 0.18.0 |
| @clerk/fastify | 0.4.8-snapshot.fe3ca7b |
| gatsby-plugin-clerk | 4.3.10-snapshot.fe3ca7b |
| @clerk/localizations | 1.18.0 |
| @clerk/nextjs | 4.21.2-snapshot.fe3ca7b |
| @clerk/clerk-react | 4.20.0 |
| @clerk/remix | 2.6.8-snapshot.fe3ca7b |
| @clerk/clerk-sdk-node | 4.10.7-snapshot.fe3ca7b |
| @clerk/shared | 0.19.0 |
| @clerk/themes | 1.7.5 |
| @clerk/types | 3.42.0 |

Tip: use the snippet copy button below to quickly install the required packages.
```sh
# @clerk/backend
npm i @clerk/backend@0.23.1-snapshot.fe3ca7b
```
```sh
# @clerk/chrome-extension
npm i @clerk/chrome-extension@0.3.9
```
```sh
# @clerk/clerk-js
npm i @clerk/clerk-js@4.47.0
```
```sh
# eslint-config-custom
npm i eslint-config-custom@0.3.0
```
```sh
# @clerk/clerk-expo
npm i @clerk/clerk-expo@0.18.0
```
```sh
# @clerk/fastify
npm i @clerk/fastify@0.4.8-snapshot.fe3ca7b
```
```sh
# gatsby-plugin-clerk
npm i gatsby-plugin-clerk@4.3.10-snapshot.fe3ca7b
```
```sh
# @clerk/localizations
npm i @clerk/localizations@1.18.0
```
```sh
# @clerk/nextjs
npm i @clerk/nextjs@4.21.2-snapshot.fe3ca7b
```
```sh
# @clerk/clerk-react
npm i @clerk/clerk-react@4.20.0
```
```sh
# @clerk/remix
npm i @clerk/remix@2.6.8-snapshot.fe3ca7b
```
```sh
# @clerk/clerk-sdk-node
npm i @clerk/clerk-sdk-node@4.10.7-snapshot.fe3ca7b
```
```sh
# @clerk/shared
npm i @clerk/shared@0.19.0
```
```sh
# @clerk/themes
npm i @clerk/themes@1.7.5
```
```sh
# @clerk/types
npm i @clerk/types@3.42.0
```

File: packages/backend/src/tokens/interstitial.ts [hunk 1]
```diff
@@ -16,6 +16,11 @@ export type LoadInterstitialOptions = {
   apiUrl: string;
   frontendApi: string;
   publishableKey: string;
+  clerkJSUrl?: string;
+  clerkJSVersion?: string;
+  /**
+   * @deprecated
```
Comment (nikosdouvlis) [inline packages/backend/src/tokens/interstitial.ts:22]: ✅

**Review by nikosdouvlis — APPROVED** [review]

File: packages/nextjs/src/server/__tests__/__snapshots__/exports.test.ts.snap [hunk 2]
```diff
@@ -7,6 +7,8 @@ exports[`/server public exports should not include a breaking change 1`] = `
   "API_VERSION",
   "AllowlistIdentifier",
   "AuthStatus",
+  "CLERK_JS_URL",
+  "CLERK_JS_VERSION",
```
Comment (dimkl) [inline packages/nextjs/src/server/__tests__/__snapshots__/exports.test.ts.snap:11]: ❓ why are those exported? are we sure about this change? Aren't those internals?
cc: @nikosdouvlis

File: packages/remix/src/ssr/utils.ts [hunk 3]
```diff
@@ -75,6 +76,10 @@ export const unknownResponse = (requestState: RequestState) => {
 };
 
 export const interstitialJsonResponse = (requestState: RequestState, opts: { loader: 'root' | 'nested' }) => {
+  console.log('HEEEE', {
+    clerkJSUrl: getEnvVariable('CLERK_JS'),
+    clerkJSVersion: getEnvVariable('CLERK_JS_VERSION'),
+  });
```
Comment (dimkl) [inline packages/remix/src/ssr/utils.ts:82]: ✂️

**Comment by clerk-cookie** [comment]
This PR has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.

**Pull request merged**